packages/application-shell/src/components/theme-switcher/new-colors-schema-announcement-dialog.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { InfoDialog } from '@commercetools-frontend/application-components';
+import { useApplicationContext } from '@commercetools-frontend/application-shell-connectors';
+import mcRebrandingAnnouncementSVG from '@commercetools-frontend/assets/images/mc-rebranding-announcement.svg';
+import Spacings from '@commercetools-uikit/spacings';
+import Text from '@commercetools-uikit/text';
+import { STORAGE_KEYS } from '../../constants';
+import messages from './messages';
+
+type TAdditionalEnvironmentValues = { enableRebrandingAnnouncement: boolean };
+type TSelectedEnvironmentValues = boolean | undefined;
+
+function useEnablebrandingAnnouncement() {
+  const enableRebrandingAnnouncement = useApplicationContext<
+    TSelectedEnvironmentValues,
+    TAdditionalEnvironmentValues
+  >((context) => context.environment.enableRebrandingAnnouncement);
+
+  return Boolean(enableRebrandingAnnouncement);
+}
+
+const NewColoursSchemaAnnouncementDialog = () => {
+  const intl = useIntl();
+  const isRebrandingAnnouncementEnabled = useEnablebrandingAnnouncement();
```

Comment:
I'm not sure I understand how the `useEnablebrandingAnnouncement` function works as it's reading a value from the environment and I think I don't get how that value is managed.

It seems what we want to know here is if the new color schema is enabled and perhaps it's a better choice to use the feature flag the same way we use in the `ThemeSwitcher` component:
```
const isRecolouringThemeEnabled = useFeatureToggle(featureFlags.RECOLOURING);
```

Reply:
updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3445/commits/2d35ba79fb6e905fef1a1c40f231e12aa9328816

Thank you

Reply:
We need to use the feature flag. Otherwise, we would start rendering the dialog before the new color schema is rolled out.

Reply:
@CarlosCortizasCT sorry I don't understand this clearly, are you saying using the feature flag directly without the `useFeatureToggle` ? 
Because it seems to me we want to check and be sure after `const isRecolouringThemeEnabled = useFeatureToggle(featureFlags.RECOLOURING);`
we can check conditionally for: `isRecolouringThemeEnabled && <NewColoursSchemaAnnouncementDialog />`
or do you mean we use `featureFlags.RECOLOURING && <NewColoursSchemaAnnouncementDialog />` directly

Reply:
Oh! Sorry. I didn't realize you moved the condition out of the dialog component to the `ThemeSwitcher`.